### PR TITLE
Hotfix/publication delay calculation

### PR DESCRIPTION
**Pull request by nightsh**
New display mode for "0 delay" trials.
image

Also, switched to internally compute the delay, as opposed to relying on spreadsheet calculations.

### CODE REVIEW

File: app/client/styles/scss/_app.scss [hunk 1]
```diff
@@ -355,6 +355,7 @@ body {
 									background-color: $green;
 									padding: $small-spacing $base-spacing 0 $base-spacing;
 									color: #fff;
+                  min-width: 200px;
```
Comment (vitorbaptista) [inline app/client/styles/scss/_app.scss]: Please fix indentation

Comment (nightsh) [inline app/client/styles/scss/_app.scss]: oh, wow, I wonder how did it end up like this 😟

Comment (nightsh) [inline app/client/styles/scss/_app.scss]: oh, it has tabs instead of spaces :-o

File: app/services/trials.js [hunk 2]
```diff
@@ -99,10 +100,6 @@ function _processData(trials) {
       return result;
     });
 
-    results = _.sortBy(results, function(item) {
-      return item.publicationDelay;
-    });
-
```
Comment (vitorbaptista) [inline app/services/trials.js]: With this removed, what's the default sorting order?

Comment (nightsh) [inline app/services/trials.js]: this wasn't affecting the order in any visible way. What should it be?

Comment (vitorbaptista) [inline app/services/trials.js]: With the current code, the sort by publication delay isn't working correctly. The expected behaviour is that the most delayed trials will appear on top (i.e. sorted in descending order for delay). I'm not sure if this code is related to the issue, but I'd start looking there.

The expected behaviour is that, when you open the page, it's sorted by delay in descending order. Also test that changing the sorting order works as expected.

**Comment by nightsh** [comment]
@vitorbaptista please take a look, I made some changes 😉

**Pull request merged**